pkgs/development/compilers/julia/build-julia-package.nix

```diff
@@ -0,0 +1,163 @@
+{ lib
+, stdenv
+, config
+, patchelf
+, julia
+, computeRequiredJuliaPackages
+, computeJuliaDepotPath
+, computeJuliaLoadPath
+, computeJuliaArtifacts
+, xvfb-run
+}:
+
+# Build an individual package
+{ fullPkgName ? "${attrs.pname}-${attrs.version}"
+
+, src
+
+, dontPatch ? false
```

Comment:
It should also be supported to disable any other phase

Reply:
I've deleted this as it's not really used. The other phases are disabled with the standard `dontConfigure`, `dontBuild`, ...